services/authorize/client.go

```diff
@@ -113,8 +123,52 @@ type AuthorizeClient interface { // nolint: golint
 	RemoveUserRoleWithContext(ctx context.Context, roleName string) error
 }
 
+func getSecret(sess *session.Session, secretsName string, out interface{}) (err error) {
```

Comment:
Refactor to new file? secret.go inside of this package. This file starting to get big

Reply:
fixed